appveyor.yml

```diff
@@ -49,7 +50,7 @@ skip_commits:
   files:
     - '*.md'
 
-notifications:
-  - provider: Slack
-    incoming_webhook:      
-      secure: 4MH9Em6TtrKalq6808dhPOqypTfYBJvVlqPaa9akNyF1h7st5qNdNezFp6T+bWXqrcZ4q/smtPcJ7LkUFHL46JDYUFlIL8FDz+ApX/vP+x0=
+#notifications:
```

Comment:
Turned that off while I was triggering many builds in my fork. When merging back, that should be uncommented again.